pages/index.js

```diff
@@ -1,9 +1,9 @@
 import Head from "next/head";
-import Content from "../components/content";
-import Header from "./components/Header";
-import Hero from "../components/hero/Hero";
+import Content from "./courses";
+import Header from "./Header";
+import Hero from "./home/Hero";
 
-export default function Home() {
+export default function Myapp() {
```

Comment:
What's the reason of changing the name of the home component? Also, components are always PascalCase, as this is suggested by React docs itself.

Reply:
Solved. I have changed the name properly. Kindly review again